web/packages/e-imports/package.json

```diff
@@ -4,7 +4,14 @@
   "description": "Enterprise-only dependencies of the Teleport Web UI",
   "private": true,
   "dependencies": {
+    "@codemirror/lang-sql": "^6.5.2",
+    "@nivo/bar": "^0.83.0",
     "@stripe/react-stripe-js": "^1.16.5",
-    "@stripe/stripe-js": "^1.48.0"
+    "@stripe/stripe-js": "^1.48.0",
+    "@uiw/codemirror-themes": "^4.21.20",
+    "@uiw/react-codemirror": "^4.21.20",
+    "d3-time-format": "^4.1.0",
+    "d3-scale": "^4.0.2",
```

Comment:
Is there any way we can update @nivo/bar and d3-scale to get rid of the vuln alert for d3-color?

https://github.com/gravitational/teleport/security/dependabot/211